components/expander.jsx

```diff
@@ -16,7 +16,7 @@ var Expander = React.createClass({
       expanded: true
     });
   },
-  onClick: function() {
+  handleClick: function(e) {
```

Comment:
renamed this handler since `onClick` could be confusing with the 'onClick' event